### DISCUSSION TITLE
登录遇到，当前网络不稳定，登录失败。推荐使用常用设备或通过手机号登录

### DISCUSSION THREAD

**Discussion by Teylusu** [simple-robot/Discussions#36] [Q&A]
title=安全提醒, message=当前网络不稳定，登录失败。推荐使用常用设备或通过手机号登录。

**Comment by ForteScarlet** [comment]
参考 [mirai#2345](https://github.com/mamoe/mirai/issues/2345) 、[mirai#2347](https://github.com/mamoe/mirai/issues/2347)
